processed 5 tasks

init:
A: object(103)

task 1 'publish'. lines 9-21:
created: object(105)
mutated: object(104)

task 2 'programmable'. lines 23-27:
created: object(107), object(108), object(109)
mutated: object(106)

task 3 'programmable'. lines 29-32:
Error: Transaction Effects Status: Invalid command argument at 0. Invalid argument to private entry function. These functions cannot take arguments from other Move functions
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidArgumentToPrivateEntryFunction }, source: None, command: Some(2) } }

task 4 'programmable'. lines 34-37:
Error: Transaction Effects Status: Invalid command argument at 0. Invalid argument to private entry function. These functions cannot take arguments from other Move functions
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidArgumentToPrivateEntryFunction }, source: None, command: Some(2) } }
